src/lib.rs

```diff
@@ -16,7 +16,7 @@
 //! * `atmega64`
 //! * `attiny85`
 #![no_std]
-#![feature(asm)]
+#![feature(asm, llvm_asm)]
```

Comment:
As in `avr-hal`, we don't need the `asm` feature anymore for now as
everything is using the old llvm-style asm syntax.